arangod/RestHandler/RestSchemaHandler.h

```diff
@@ -0,0 +1,87 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2025-2025 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Business Source License 1.1 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://github.com/arangodb/arangodb/blob/devel/LICENSE
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Koichi Nakata
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "RestHandler/RestCursorHandler.h"
+#include "Graph/GraphManager.h"
+
+#include <set>
+
+namespace arangodb {
+namespace aql {
+class QueryRegistry;
+}
+namespace rest {
+
+class RestSchemaHandler : public RestCursorHandler {
+ public:
+  RestSchemaHandler(ArangodServer& server, GeneralRequest* request,
+                    GeneralResponse* response,
+                    aql::QueryRegistry* queryRegistry);
+
+  RestStatus execute() override;
+  RestStatus handleQueryResult() override;
+
+ private:
+  static constexpr uint64_t defaultSampleNum = 100;
+  static constexpr uint64_t defaultExampleNum = 1;
+  static const std::string queryStr;
+  graph::GraphManager _graphManager;
+
+  Result lookupSchema(uint64_t sampleNum, uint64_t exampleNum);
```

Comment:
Could you add brief descriptions what all those method should do?
1 - 2 Sentences is enough.